EnglishTranslation-master/SKILL.tsv

```diff
@@ -4989,7 +4989,7 @@ SKILL_20150729_004988	Heal: Remove Damage
 SKILL_20150729_004989	[Heal] will not inflict damage and react to enemies.
 SKILL_20150729_004990	Increases the range of [Melstis] to 80.
 SKILL_20150729_004991	Increases the maximum number of zombies from [Zombify] by 1.
-SKILL_20150730_004992	Summons a devil from the card placed in the Grimoire. The attack and defense of the summoned devil is affected by your INT and SPR. Increases the abilities by 10% proportional to the amount of stars of the card. Only Devil-type cards can be placed in the Grimoire.
+SKILL_20150730_004992	Summon a devil from the card placed in the Grimoire. The attack and defense of the summoned devil is affected by your INT and SPR. Increases the stats by 10% proportional to the amount of stars of the card. Only Devil-type cards can be placed in the Grimoire.
```

Comment:
I can't really tell how the stats are increasing... is it 10%(multiplied by # of stars) of your INT and SPR?
If so, maybe the description should be
` The attack and defense of the summoned devil is affected by 10% your INT and SPR, proportional to the amount of stars of the card. `
But if it means the stats in general are increased by the stars on the card, then maybe all that should be changed is
 `The stats increase by 10% proportional to the amount of stars of the card.`